requirements.txt

```
@@ -1,3 +1,4 @@
 sphinx
 sphinx_rtd_theme
 recommonmark
+sphinxcontrib-mermaid
```

```suggestion
sphinxcontrib-mermaid
six
```

This made it build for me

---

Fixed in b2c7a43f008e84ee16baddf6cf42a00189659710